Microsoft.Toolkit.Uwp.UI.Controls/CameraPreview/CameraPreview.xaml

```diff
@@ -0,0 +1,27 @@
+﻿<ResourceDictionary
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:Microsoft.Toolkit.Uwp.UI.Controls">
+    <Style x:Name="MessageStyle" TargetType="TextBlock">
+        <Setter Property="FontSize" Value="14"/>
+        <Setter Property="FontWeight" Value="Bold"/>
+        <Setter Property="Foreground" Value="Red"/>
+    </Style>
+    <Style TargetType="local:CameraPreview" >
+        <Setter Property="Template">
+            <Setter.Value>
+                <ControlTemplate TargetType="local:CameraPreview">
+                    <Grid Background="{TemplateBinding Background}">
+                        <MediaPlayerElement x:Name="MediaPlayerElementControl" MinWidth="300" Width="400" HorizontalAlignment="Left">
+                        </MediaPlayerElement>
+                        <Button x:Name="FrameSourceGroupButton" Background="Transparent" 
```

Comment:
This button should only be visible when there are more than one frame source. There should also be a property to hide the button by the developer.

Reply:
Updated per comment.